integration/tests/test_placement.py

```diff
@@ -0,0 +1,72 @@
+import dcos.http
+import pytest
+import shakedown
+
+from tests.test_utils import (
+    DEFAULT_BROKER_COUNT,
+    PACKAGE_NAME,
+    check_health,
+    get_dcos_command,
+    get_kafka_command,
+    install,
+    marathon_api_url_with_param,
+    request,
+    spin,
+    uninstall,
+    get_and_verify_plan,
+)
+
+
+TOPIC_NAME = 'topic1'
+NUM_TEST_MSGS = 24
+
+
+def setup_module(module):
+    uninstall()
+
+
+def teardown_module(module):
+    uninstall()
+
+
+
+@pytest.mark.placement
+@pytest.mark.sanity
+def test_marathon_unique_hostname():
+    install(additional_options = {'service':{'placement_constraint':'hostname:UNIQUE'}})
+    check_health()
+    plan = infinity_commons.get_and_verify_plan(lambda p: p['status'] == 'COMPLETE')
+    assert plan['status'] == 'COMPLETE'
+    uninstall()
+
+
+# effectively same as 'hostname:UNIQUE':
+@pytest.mark.placement
+@pytest.mark.sanity
+def test_marathon_max_one_per_hostname():
+    install(additional_options = {'service':{'placement_constraint':'hostname:MAX_PER:1'}})
+    check_health()
+    plan = infinity_commons.get_and_verify_plan(lambda p: p['status'] == 'COMPLETE')
+    assert plan['status'] == 'COMPLETE'
+    uninstall()
+
```

Comment:
Python style wants two blank lines between functions (should add an extra newline here)